README.md

```diff
@@ -1 +1,64 @@
-# sticky-egret
\ No newline at end of file
+## Description
+
+Make a scalable, single page blog app, and make it responsive, BEAUTIFUL and/or INNOVATIVE!
+
+
+## Context
+
+Learners research award winning, innovative blog designs and use wireframing to pre-visualize their app and put some thought into their desired user experience. Learners will gain practice templating to render blog posts as different page views of a single page app. Learners will gain practice setting up a server using Express and/or Express-generator, and templating using Pug (or Handlebars, or equivalent).
+
+Creating this web application will provide exposure to:
+- Express (or other web framework)
+- Javascript
+- Use PostgreSQL and Postico relational database interactions (Create, Read, Update, Delete), with SQL practice
+- Use PUG
+- Use RESTeasy or Postman to test routes
+- React as a stretch goal
+
+
+## Specifications (100% total) [Github](https://github.com/eobaah/guarded-porcupine)
+- [ ] Wireframe of app that puts thought into excellent user experience to be implemented. Research existing blog websites that look and work great, or that are totally out-of-the-box design wise, then sketch (by hand or use a tool like Balsamiq ) your app's look and functionality, page by page. Have fun making it yours! (2.5%)
+- [ ] Optimize your assets on the front-end (your images should be web-ready, for example) (2.5%)
+- [ ] Use responsive design principles (make it mobile first!)(5%)
+
+User should be able to: (10%)
+- [ ] See an excerpt of all or a selection of the latest blog posts on front/home page - excerpt may include blog post title, photo, description or except of main text, etc.
+- [ ] Click on a specific blog post except to be taken to the full blog post.
```

Comment:
What are the features/functionalities of a full blog post?  What should the user see?  Are there comments?